data/objects/can.urdf

```diff
@@ -1,18 +1,15 @@
 <robot name="can">
   <link name="base_link">
     <collision>
-      <origin xyz="0.0 0.0 0.0615" />
+      <origin xyz="-0.3 0.0 0.0" />
       <geometry>
         <cylinder length="0.123" radius="0.033" />
       </geometry>
     </collision>
     <visual>
-      <origin xyz="0.0 0.0 0.0615" />
-      <material name="c">
-        <color rgba="0.65 0.7 0.7 1.0" />
-      </material>
+      <origin xyz="-0.3 0.0 0.0" />
```

Comment:
I guess this origin is necessary because of the offset of the visual mesh. With this offset, is the center of the visual mesh will be placed at (0, 0, 0) in the local frame?

Also, the origin of collision shape shouldn't be changed because we keep using the primitive cylinder.